src/test/java/test_login_use_case/TestLoginUseCase.java

```diff
@@ -0,0 +1,116 @@
+
+        loginPresenter testingPresenter = new loginPresenter() {
+            @Override
+            public loginResponseModel prepareSuccessView(loginResponseModel responseModel) {
+                assertEquals("demetriusjr", responseModel.getCommonUser().getName());
+                assertTrue(testingDatabase.hasKey("demetriusjr"));
+                testingDatabase.deleteStorageFile();
+                return null;
+            }
+
+            @Override
+            public loginResponseModel prepareFailView(String error) {
+                assertEquals(error, "Account does not exist");
+                return null;
+            }
+        };
+
+        FridgeFactory testingFridgeFactory = new CommonFridgeFactory();
+        ArrayList<CommonIngredient> ingredients = new ArrayList<>();
+        CommonFridge testFridge = (CommonFridge) testingFridgeFactory.create(ingredients);
+        UserFactory testingUserFactory = new CommonUserFactory();
+        CommonUser testUser = (CommonUser) testingUserFactory.create("demetriusjr", testFridge);
+        StorageObjects objects = new StorageObjects(testUser, testFridge);
+        testingDatabase.store("demetriusjr", objects);
+
+        loginInputBoundary testing_login = new LoginInteractor(testingDatabase, testingPresenter);
+        //loginResponseModel inputData = new loginResponseModel(testFridge ,testUser);
+        loginRequestModel inputData = new loginRequestModel("demetriusjr");
+        testing_login.create(inputData);
+    }
+
+    /**
+     * Check if user does not exist program gives expected result
+     */
+    @Test
+    void nonexistentTest(){
+
+        DatabaseGateway testingDatabase = new Database("Test_Storage");
+
+        loginPresenter testingPresenter = new loginPresenter() {
+            @Override
+            public loginResponseModel prepareSuccessView(loginResponseModel responseModel) {
+                assertEquals("demetriusjr", responseModel.getCommonUser().getName());
+                assertTrue(testingDatabase.hasKey("demetriusjr"));
+                testingDatabase.deleteStorageFile();
+                return null;
+            }
+
+            @Override
+            public loginResponseModel prepareFailView(String error) {
+                assertEquals(error, "Account does not exist");
+                return null;
+            }
+        };
+        loginInputBoundary testing_login = new LoginInteractor(testingDatabase, testingPresenter);
+        loginRequestModel inputData = new loginRequestModel("demetriusSr");
+        testing_login.create(inputData);
+
+
+    }
+
+    /**
+     * Expected Result when nothing is entered
+     */
+    @Test
+    void nothingTest(){
+        DatabaseGateway testingDatabase = new Database("Test_Storage");
+        loginPresenter testingPresenter = new loginPresenter() {
+            @Override
+            public loginResponseModel prepareSuccessView(loginResponseModel responseModel) {
+                return null;
```

Comment:
I think you should add an assertion statement that fails here. If a future change accidentally makes login succeed for no username entered, the prepareSuccessView() function would run and the test would pass. The tests for Registration may be affected too.